lib/list.js

```diff
@@ -48,114 +65,62 @@ function validate (opts) {
   }
 }
 
-function defineCollectionUrl (opts) {
-  const { collection, category } = opts;
-  const baseUrl = `${BASE_URL}/store/apps`;
-  const path = collection.startsWith('NEW_') ? `${baseUrl}/new` : `${baseUrl}/top`;
-  return (category)
-    ? `${path}/category/${category}`
-    : path;
-}
-
 function buildInitialUrl (opts) {
-  const url = defineCollectionUrl(opts);
-
-  // For the initial collection lookup we force the language to english
-  // since we depend on collection names to identify the proper cluster later on
   const queryString = {
-    hl: 'us',
+    hl: opts.lang,
     gl: opts.country
   };
 
   if (opts.age) {
     queryString.age = opts.age;
   }
+  const url = `https://play.google.com/_/PlayStoreUi/data/batchexecute?rpcids=vyAe2&source-path=%2Fstore%2Fapps&f.sid=-4178618388443751758&bl=boq_playuiserver_20220612.08_p0&authuser=0&soc-app=121&soc-platform=1&soc-device=1&_reqid=82003&rt=c`;
```

Comment:
@mmoksh should use ${BASE_URL} here as well.

@facundoolano will you approve a PR where the baseURL can be passed as a parameter, so we can use services like scraperstack?

Reply:
+1 to using BASE_URL here

@nadavzipo I would approve but that should be done in a separate PR after this one gets merged

Reply:
```suggestion
  const url = `${BASE_URL}/_/PlayStoreUi/data/batchexecute?rpcids=vyAe2&source-path=%2Fstore%2Fapps&f.sid=-4178618388443751758&bl=boq_playuiserver_20220612.08_p0&authuser=0&soc-app=121&soc-platform=1&soc-device=1&_reqid=82003&rt=c`;
```